### PR TITLE
Removing old training scripts and website support - #1534 #1538

### PR DESCRIPTION
**Pull request by tilmankamp**

### CODE REVIEW

File: bin/run-tc-ldc93s1_frozen.sh [hunk 1]
```diff
@@ -1,23 +0,0 @@
-#!/bin/sh
```
Comment (lissyx) [inline bin/run-tc-ldc93s1_frozen.sh]: We probably still want this ?

File: bin/run-tc-ldc93s1_new.sh [hunk 2]
```diff
@@ -1,22 +0,0 @@
-#!/bin/sh
```
Comment (lissyx) [inline bin/run-tc-ldc93s1_new.sh]: We probably still want this one as well ?

File: bin/run-ldc93s1.sh [hunk 3]
```diff
@@ -1,29 +0,0 @@
-#!/bin/sh
```
Comment (lissyx) [inline bin/run-ldc93s1.sh]: This one was useful for debugging people's setting up issues

Comment (reuben) [reply bin/run-ldc93s1.sh]: Yes, please keep run-ldc93s1.sh, I use it for development all the time.

**Comment by tilmankamp** [comment]
Re-added all files @lissyx mentioned.

**Pull request merged**